Readme.md

```diff
@@ -6,30 +6,111 @@
 
 ## Installation
 
+When evaluating this application and you do not require a production-grade setup, we highly recommend using the Docker compose installation method in development mode. Why development mode? Because it allows you to skip deploying the authentication stack to AWS :). 
+### Docker
+
+There are 2 Docker compose files available:
+
+- docker-compose.yml
+
+Production mode, authentication will happen via JWTs provided by AWS Cognito. See the folder `infra/auth` 
+for details on how its set up
+
+`docker-compose up -d`
+
+- docker-compose-dev.yml
+
+Development mode, any calls to the API are automatically authenticated
+This does mean only 1 user can exist in the database at any time, it might get lonely in your solo squad :(
+
+`docker-compose -f docker-compose-dev.yml up -d`
+
 ### Server
 
+We developed and tested on Linux systems, YMMV on other operating systems...
+
 ```sh
 docker-compose up -d 
 cd server
 npm ci # Use "ci" so it respects the lockfile
 
 # When running "npm start" you will be running in production mode
-# This means, authentication will happen via JWTs provded by AWS Cognito
-# See the folder infra/auth for details on how its set up
 npm start
 
 # When running "npm run dev" you will be running in development mode
-# Any calls to the API are automatically authenticated
-# This does mean only 1 user can exist in the database at any time
-# It might get lonely in your solo squad :(
 npm run dev
 ```
 
+
 ### Client
 
-// TODO: Emiel
+We developed and tested on Linux systems, YMMV on other operating systems...
+
+
+```sh
+cd client
+npm ci # Use "ci" so it respects the lockfile
+
+# Some weird dependency issue... // TODO: Emiel explain better? :D
+export SKIP_PREFLIGHT_CHECK=true
```

Comment:
Should be fixed, 

Maybe set, this as extra if while npm installing shows this error, add ... in `.env`. 
Other solution (and should actually be tried first) is npm i --legacy-peer-deps